### ISSUE TITLE
Missing permissions to list Nodes

### ISSUE THREAD

**Issue by mzeevi** [dana-team/hns-nqs-plugin#11]
Missing a kubebuilder market it seems

```
2023-07-14T16:12:02Z	INFO	controller-runtime.metrics	Metrics server is starting to listen	{"addr": "127.0.0.1:8080"}
2023-07-14T16:12:02Z	INFO	setup	starting manager
2023-07-14T16:12:02Z	INFO	Starting server	{"kind": "health probe", "addr": "[::]:8081"}
2023-07-14T16:12:02Z	INFO	starting server	{"path": "/metrics", "kind": "metrics", "addr": "127.0.0.1:8080"}
I0714 16:12:02.157337       1 leaderelection.go:245] attempting to acquire leader lease nodequotasync-system/3185cb04.hns.io...
I0714 16:12:20.112944       1 leaderelection.go:255] successfully acquired lease nodequotasync-system/3185cb04.hns.io
2023-07-14T16:12:20Z	DEBUG	events	nodequotasync-controller-manager-7c6447cbbc-h52dw_5b6dbb9e-31ab-451f-9b04-1371f691e4cd became leader	{"type": "Normal", "object": {"kind":"Lease","namespace":"nodequotasync-system","name":"3185cb04.hns.io","uid":"52b2f134-d112-40f1-834f-c6f118337ece","apiVersion":"coordination.k8s.io/v1","resourceVersion":"190195945"}, "reason": "LeaderElection"}
2023-07-14T16:12:20Z	INFO	Starting EventSource	{"controller": "nodequotaconfig", "controllerGroup": "dana.hns.io", "controllerKind": "NodeQuotaConfig", "source": "kind source: *v1alpha1.NodeQuotaConfig"}
2023-07-14T16:12:20Z	INFO	Starting EventSource	{"controller": "nodequotaconfig", "controllerGroup": "dana.hns.io", "controllerKind": "NodeQuotaConfig", "source": "kind source: *v1.Node"}
2023-07-14T16:12:20Z	INFO	Starting Controller	{"controller": "nodequotaconfig", "controllerGroup": "dana.hns.io", "controllerKind": "NodeQuotaConfig"}
W0714 16:12:20.120478       1 reflector.go:533] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
E0714 16:12:20.120528       1 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
W0714 16:12:21.447961       1 reflector.go:533] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
E0714 16:12:21.448003       1 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
W0714 16:12:24.405808       1 reflector.go:533] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
E0714 16:12:24.405846       1 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
W0714 16:12:27.948925       1 reflector.go:533] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
E0714 16:12:27.948969       1 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
W0714 16:12:35.481564       1 reflector.go:533] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
E0714 16:12:35.481602       1 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
W0714 16:12:58.228161       1 reflector.go:533] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
E0714 16:12:58.228207       1 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.2/tools/cache/reflector.go:231: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:nodequotasync-system:nodequotasync-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
****
```

/kind bug

**Comment by mzeevi** [comment]
closed by #9

**Comment by mzeevi** [comment]
actually still missing
